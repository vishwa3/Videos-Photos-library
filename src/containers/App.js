import React, { Component } from 'react';
import Header from '../common/Header';
import HomePage from '../components/HomePage';
// The parent component renders the Header component and component(s) in the
// route the user navigates to.
class App extends Component {
  render() {
    console.log("in App");
    return (
      <div className="container-fluid text-center">
        <Header />
        <HomePage />
      </div>
    );
  }
}


export default App;