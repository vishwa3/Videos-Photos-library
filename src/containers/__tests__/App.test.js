import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import App from '../App';

describe('App',() => {
  test('snapshot for App', () => {
    const component = renderer.create(<Router><App /></Router>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })
})