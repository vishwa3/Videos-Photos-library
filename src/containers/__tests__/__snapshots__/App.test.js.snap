// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot for App 1`] = `
<div
  className="container-fluid text-center"
>
  <div
    className="jumbotron center"
  >
    <h1
      className="lead"
      style={
        Object {
          "fontFamily": "Pacifico",
        }
      }
    >
      Welcome to Media Library built with React, Redux, and Redux-saga 
    </h1>
    <div>
      <a
        href="/library"
        onClick={[Function]}
      >
        <button
          className="btn btn-lg btn-primary"
        >
           Visit Library
        </button>
      </a>
    </div>
  </div>
</div>
`;
